src/applications/claims-status/tests/e2e/05.claim-document-request.cypress.spec.js

```diff
@@ -158,154 +158,226 @@ describe('When feature toggle cst_5103_update_enabled enabled', () => {
   });
 });
 
-describe('Type 1 Error Alert', () => {
-  context('when cst_show_document_upload_status is disabled', () => {
-    it('should display the old alert and not the new type 1 unknown error alert', () => {
-      setupPageAndIntercepts(false);
-      setupUnknownErrorMock();
-      uploadFileWithDocType('test-document.txt');
-      clickSubmitFilesButton(false);
+describe('Document upload', () => {
```

Comment:
Since I added some success scenarios, I adjusted the describe blocks accordingly.